DXRMissions/DeusEx/Classes/DXRMissionsM06.uc

```diff
@@ -105,7 +191,7 @@ function DeleteGoal(Goal g, GoalLocation Loc)
 
 function GenerateDTSHintCube(Goal g, GoalLocation Loc)
 {
-    SpawnDatacube(vectm(-1857.841064, -158.911865, 2051.345459), rotm(0,0,0),
+    SpawnDatacubePlaintext(vectm(-1857.841064, -158.911865, 2051.345459), rotm(0,0,0),
         "I borrowed the sword but forgot it somewhere...  Maybe "$Loc.name$"?", true);
```

Comment:
what if instead of telling you the exact location, it could be like "Maybe A or B" where one is the actual location and the other is a random incorrect location lol